x/ecocredit/module/module.go

```diff
@@ -206,14 +249,20 @@ func (Module) RandomizedParams(r *rand.Rand) []simtypes.ParamChange {
 }
 
 // RegisterStoreDecoder registers a decoder for ecocredit module's types
-func (Module) RegisterStoreDecoder(sdr sdk.StoreDecoderRegistry) {
-}
+func (Module) RegisterStoreDecoder(_ sdk.StoreDecoderRegistry) {}
 
 // WeightedOperations returns all the ecocredit module operations with their respective weights.
 // NOTE: This is no longer needed for the modules which uses ADR-33, ecocredit module `WeightedOperations`
 // registered in the `x/ecocredit/server` package.
-func (Module) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
-	return nil
+func (a Module) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
```

Comment:
can you also update godoc?

Reply:
https://github.com/regen-network/regen-ledger/pull/1357/commits/90d36028d377ef2272e3bd2e339f63f34769d677 👍🏻